pages/windows/stop-service.md

```diff
@@ -0,0 +1,17 @@
+# Stop-Service
+
+> Stops one or more running services.
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.management/stop-service>.
+
+- Stop a service on the local computer:
+
+`Stop-Service -Name {{"sysmonlog"}}`
```

Comment:
```suggestion
`Stop-Service -Name {{service_name}}`
```

Reply:
Following the other suggestions would you rather user the following?

```suggestion
`Stop-Service -Name "{{service_name}}"`
```

Reply:
I'm not sure if the service names usually come with spaces. I think the user will figure it out anyway

Reply:
ok